packages/portal-proto/src/features/GenomicTables/GenesTable/TableComponents/GenesTableCohort.tsx

```diff
@@ -48,8 +48,6 @@ const GenesTableCohort = ({
       tooltip={
         isDemoMode
           ? "Feature not available in demo mode"
-          : toggledGenes.includes(geneID)
```

Comment:
Why was this removed?

Reply:
good catch, placing it back. I know i was removing similar from aria labels, don't know why this was changed as its a tooltip